test/testdata/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: template-service-broker
+parameters:
+- name: IMAGE
+  value: openshift/origin:latest
+- name: TSB_TEMPLATE_NAMESPACE
+  value: openshift
+- name: NAMESPACE
+  value: openshift-template-service-broker
+- name: KUBE_SYSTEM
+  value: kube-system
+objects:
+
+# to create the tsb server
+- apiVersion: extensions/v1beta1
+  kind: DaemonSet
+  metadata:
+    namespace: ${NAMESPACE}
+    name: apiserver
+    labels:
+      apiserver: "true"
+  spec:
+    template:
+      metadata:
+        name: apiserver
+        labels:
+          apiserver: "true"
+      spec:
```

Comment:
> parameterised nodeselector needed?

I think I'd leave it to be controlled by the namespace annotations to avoid being prescriptive about deployment topology.